### PR TITLE
Add agility course tooltips and indicate rooftop courses in worldmap

### LINKED ISSUE #11426: Show agility course names on world map, and mark Rooftop courses differently
Currently, the world map display agility training instead of Draynor Village Rooftop Course
Could you please add that?

Screenshot 2020-05-01 at 20 06 55

also, agility courses that are rooftop can they have this icon on the worldmap 
Screenshot 2020-05-01 at 20 08 45 instead of Screenshot 2020-05-01 at 20 09 04

Screenshot 2020-05-01 at 20 11 03

### PR DESCRIPTION
**Pull request by melkypie**
Closes #11426

### CODE REVIEW

**Comment by deathbeam** [comment]
Needs rebase

**Comment by melkypie** [comment]
Rebased

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/AgilityCourseLocation.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+	AgilityCourseLocation(String tooltip, WorldPoint location)
+	{
+		this.tooltip = tooltip;
+		this.location = location;
+	}
+
+	public boolean isRooftopCourse()
+	{
+		return this.name().contains("ROOFTOP_COURSE");
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/AgilityCourseLocation.java:63]: This logic can be moved to the constructor and stored to an additional `rooftopCourse` field.

**Comment by melkypie** [comment]
Made the requested change and added missing gnome stronghold course

**Comment by melkypie** [comment]
> LGTM. Needs rebase

Rebased

File: runelite-client/src/main/java/net/runelite/client/plugins/worldmap/AgilityCourseLocation.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	BARBARIAN_OUTPOST_AGILITY_COURSE("Barbarian Outpost Agility Course", new WorldPoint(2544, 3569, 0)),
+	BRIMHAVEN_AGILITY_ARENA("Brimhaven Agility Arena", new WorldPoint(2806, 3193, 0)),
+	CANIFIS_ROOFTOP_COURSE("Canifis Rooftop Course", new WorldPoint(3506, 3490, 0)),
+	DRAYNOR_VILLAGE_ROOFTOP_COURSE("Draynor Village Rooftop Course", new WorldPoint(3103, 3279, 0)),
+	FALADOR_ROOFTOP_COURSE("Falador Rooftop Course", new WorldPoint(3035, 3342, 0)),
+	GNOME_STRONGHOLD_AGILITY_COURSE("Gnome Stronghold Agility Course", new WorldPoint(2474, 3436, 0)),
+	PENGUIN_AGILITY_COURSE("Penguin Agility Course", new WorldPoint(2638, 4041, 0)),
+	POLLNIVNEACH_ROOFTOP_COURSE("Pollnivneach Rooftop Course", new WorldPoint(3350, 2963, 0)),
+	PRIFDDINAS_AGILITY_COURSE("Prifddinas Agility Course", new WorldPoint(3253, 6109, 0)),
+	RELLEKKA_ROOFTOP_COURSE("Rellekka Rooftop Course", new WorldPoint(2624, 3677, 0)),
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/worldmap/AgilityCourseLocation.java:47]:
```suggestion
	RELEKKA_ROOFTOP_COURSE("Relekka Rooftop Course", new WorldPoint(2624, 3677, 0)),
```

Comment (melkypie) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/AgilityCourseLocation.java:47]: It's actually with 2 L's. https://oldschool.runescape.wiki/w/Rellekka

Comment (Nightfirecat) [reply runelite-client/src/main/java/net/runelite/client/plugins/worldmap/AgilityCourseLocation.java:47]: D'oh. That's what I get for assuming that a quick search gave me the correct spelling.

**Comment by melkypie** [comment]
Rebased again

**Pull request merged**